src/Security/Authorization/Core/src/DenyAnonymousAuthorizationRequirement.cs

```diff
@@ -9,6 +9,8 @@ namespace Microsoft.AspNetCore.Authorization.Infrastructure;
 /// <summary>
 /// Implements an <see cref="IAuthorizationHandler"/> and <see cref="IAuthorizationRequirement"/>
 /// which requires the current user must be authenticated.
+/// This calls <see cref="AuthorizationHandlerContext.Succeed"/> for authenticated users. Like all built-in requirements, it never calls <see cref="AuthorizationHandlerContext.Fail"/>.
+/// The <see cref="DefaultAuthorizationEvaluator"/> produces a failed <see cref="AuthorizationResult" /> when any requirement has not succeeded even if other requirements have succeeded and no requirement has explicitly failed.
```

Comment:
```suggestion
/// Implements an <see cref="IAuthorizationHandler"/> and <see cref="IAuthorizationRequirement"/> which requires the current user must be authenticated.
/// This calls <see cref="AuthorizationHandlerContext.Succeed(IAuthorizationRequirement)"/> for authenticated users. Like all built-in requirements,
/// it never calls <see cref="AuthorizationHandlerContext.Fail()"/>. The <see cref="DefaultAuthorizationEvaluator"/> produces a failed <see cref="AuthorizationResult" /> 
/// when any requirement has not succeeded even if other requirements have succeeded, and no requirement has explicitly failed.
```

Hopefully one last try.